Assets/Scripts/Framework/SceneSwitcher.cs

```diff
@@ -19,16 +19,30 @@ private void Awake()
         /// </summary>
         public void LoadScene() => SceneManager.LoadScene(sceneToLoad);
 
+        /// <summary>
+        /// Set the sceneToLoad property to a new scene, if this succeeds it will load it, otherwise it will give an error.
+        /// </summary>
+        /// <param name="targetScene"></param>
+        public void SetAndLoadScene(string targetScene)
+        {
+            if(SetSceneToLoad(targetScene))
+                LoadScene();
+        }
+
         /// <summary>
         /// Set the sceneToLoad property to a new scene, if it's not in the build settings you will get an error.
         /// </summary>
         /// <param name="targetScene"></param>
```

Comment:
Comment voor param name is hier ook leeg >:(
Mist ook comment voor return value